shared/build.gradle.kts

```diff
@@ -11,6 +15,28 @@ plugins {
 kotlin {
     androidTarget()
     jvm()
+    
+    wasmJs {
+        binaries.executable()
+        browser { 
+
+        }
+    }
+
+    js {
+        binaries.executable()
+        browser { 
+            commonWebpackConfig {
+                val out = this.output 
```

Comment:
Please log an issue and add link in comment

Reply:
It's not a bug, it was required for dynamic loading.

Removed, since right now it's not needed, likely will return later.